src/settings.cpp

```diff
@@ -41,7 +41,8 @@ settings::settings()
     bip9_version_base(0x20000000),
     satoshi_per_bitcoin(100000000),
     initial_block_subsidy_bitcoin(50),
-    recursive_money(9999999989u)
+    recursive_money(max_money_recursive(bitcoin_to_satoshi(
+        initial_block_subsidy_bitcoin)))
```

Comment:
This can’t be safely exposed as a property unless computed (vs. initialized). Same holds for other compued initializatuons above.